source/ProtonPack/ProtonPack.ino

```diff
@@ -3031,27 +3070,18 @@ void wandFiring() {
   ms_firing_length_timer.start(i_firing_timer_length);
 }
 
-void wandStoppedFiring() {
-  // Stop all other firing sounds.
-  wandStopFiringSounds();
-  ms_firing_sound_mix.stop();
-
-  serial1Send(A_FIRING_STOPPED);
-
-  // Stop the auto speed timer.
-  ms_cyclotron_auto_speed_timer.stop();
-
-  // Adjust the gain with the Afterlife idling track.
-  if(i_mode_year == 2021 && i_wand_power_level < 5) {
-    if(ms_idle_fire_fade.remaining() < 1000) {
-      adjustGainEffect(S_AFTERLIFE_PACK_IDLE_LOOP, i_volume_effects, true, 30);
-    }
-    else {
-      adjustGainEffect(S_AFTERLIFE_PACK_IDLE_LOOP, i_volume_effects, true, ms_idle_fire_fade.remaining());
+void modeFireStopSounds() {
```

Comment:
I presume this allows for a small delay to the wavTrigger so that the communications from the wand and pack can have time to be received and take action?

Reply:
So I found during testing if I spread out several of the serial commands with tiny delays, it makes the system more stable during the fastled updates during any button smashing.